pkg/virt-launcher/virtwrap/network/podinterface_test.go

```diff
@@ -90,17 +85,9 @@ var _ = Describe("Pod Network", func() {
 	var mtu int
 	var cacheFactory cache.InterfaceCacheFactory
 	var libvirtUser string
-	var newPodNIC = func(vmi *v1.VirtualMachineInstance) podNIC {
-		return podNIC{
-			cacheFactory: cacheFactory,
-			handler:      mockNetwork,
-			vmi:          vmi,
-		}
-	}
-	var createDefaultPodNIC = func(vmi *v1.VirtualMachineInstance) podNIC {
-		podnic := newPodNIC(vmi)
-		podnic.vmiSpecIface = &vmi.Spec.Domain.Devices.Interfaces[0]
-		podnic.vmiSpecNetwork = &vmi.Spec.Networks[0]
+	var createDefaultPodNIC = func(vmi *v1.VirtualMachineInstance) *podNIC {
+		podnic, err := newPodNIC(vmi, &vmi.Spec.Networks[0], mockNetwork, cacheFactory, nil)
```

Comment:
do you mind passing the mockNetowrk and cacheFactory as parameters to the function ? If I want to use a table and pass this pod nic thing as an argument it'll explode on my face, since these arguments are not initialized.